fluster/test.py

```diff
@@ -97,15 +97,24 @@ def _test(self) -> None:
             self.test_suite.test_vectors[
                 self.test_vector.name
             ].test_result = TestVectorResult.FAIL
-            if self.test_vector.result.lower() == result.lower():
+            test_vector_result = self.test_vector.result.lower()
+            if test_vector_result == result.lower() or result == "True":
```

Comment:
Do we need to check `"True"` str with last version?

Reply:
If I don't leave True, the RGBA version won't work, if it works one day

Reply:
"True" is a value that comes from Javascript? Can we change it to be something more meaningful?

Reply:
is eliminated